gcp/modules/gcp-stackdriver-lbm/main.tf

```diff
@@ -48,3 +48,40 @@ resource "null_resource" "apply_stackdriver_lbm" {
     EOF
   }
 }
+
+resource "null_resource" "destroy_stackdriver_lbm" {
+  provisioner "local-exec" {
+    when = "destroy"
+
+    command = <<EOF
+      export PROJECT_ID=${var.project_id}
+      export GOOGLE_CLOUD_KEYFILE=${var.serviceaccount_key}
+
+      # To avoid race condition with gcp-stackdriver-monitoring module
+      sleep 30
```

Comment:
How about a message here so the viewer knows this module isn't stuck?

Reply:
Done.